history/states/889-Elisabethville.txt

```diff
@@ -1,50 +1,45 @@
-state = {
-	id = 889
-	name = "STATE_889"
-	manpower = 708190
-	state_category = pastoral
-
-	resources={
-		tungsten = 2
-	}
-
-	history = {
-		owner = COG
-
-		IF = {
-			limit = {
-				AND = {
-					has_dlc = "Gotterdammerung"
-					has_game_rule = {
-						rule = BEL_focus_tree_selection
-						option = STANDARD_TREE
-					}
-				}
-			}
-			add_dynamic_modifier = {
-				modifier = COG_state_loyal_to_free_congo_modifier
-			}
-		}
-
-		buildings = {
-			infrastructure = 1
-
-		}
-		victory_points = {
-			191 3
-		}
-		victory_points = {
-			1516 1
-		}
-		add_core_of = COG
-		add_core_of = KAT
-
-	}
-	
-	provinces = {
-		191 1516 1817 1968 4515 12795 12907 
-	}
-
-	buildings_max_level_factor = 1.000
-	local_supplies = 0.000
-}
+state = {
+	id = 889
+	name = "STATE_889"
+	manpower = 708190
+	state_category = pastoral
+
+	resources = {
+		tungsten = 2.000
+		coal = 1
+	}
+
+	history = {
+
+		owner = BEL
+
+		IF = {
+			limit = {
+				has_dlc = "Gotterdammerung"
+			}
+			transfer_state_to = COG
+			add_dynamic_modifier = {
+				modifier = COG_state_loyal_to_free_congo_modifier
+			}
+		}
+
+		buildings = {
+			infrastructure = 1
+		}
+
+		add_core_of = COG
+		add_core_of = KAT
+
+		victory_points = {
+			191 2 # Elizabehville
+		}

```

Comment:
It seems you removed one of the victory point? Is that intentional? In general this diff is broken

Reply:
The other victory point I mean, 1516